auth/saml/token.go

```diff
@@ -0,0 +1,199 @@
+// Copyright 2015 tsuru authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package saml
+
+import (
+	"crypto"
+	"crypto/rand"
+	"errors"
+	"fmt"
+	"time"
+
+	"github.com/tsuru/config"
+	"github.com/tsuru/tsuru/auth"
+	"github.com/tsuru/tsuru/db"
+	"github.com/tsuru/tsuru/permission"
+
+	"gopkg.in/mgo.v2"
+	"gopkg.in/mgo.v2/bson"
+)
+
+const (
+	keySize           = 32
+	defaultExpiration = 7 * 24 * time.Hour
+)
+
+var (
+	tokenExpire time.Duration
+)
```

Comment:
You don't need the parenthesis for grouping here, as it's a single variable. Please do:

```go
var tokenExpire time.Duration
```